apps/backend/src/app/api/latest/cron/send-email-digest/route.ts

```diff
@@ -0,0 +1,92 @@
+import { getEmailConfig, sendEmail, sendEmailFromTemplate } from "@/lib/emails";
+import { getSoleTenancyFromProject } from "@/lib/tenancies";
+import { prismaClient } from "@/prisma-client";
+import { Project } from "@prisma/client";
+import { getEnvVariable } from "@stackframe/stack-shared/dist/utils/env";
+
+export async function GET(request: Request) {
```

Comment:
make this a SmartRouteHandler (see eg. /api/v1 for a simple route that just uses SmartRouteHandler). generally, everything should be a SmartRouteHandler

Reply:
Completed